Dockerfile

```diff
@@ -7,7 +7,7 @@
 
 # Build the CSL WebAssembly module
 # This stage of the build also copies the wasm_exec.js script to the frontend source folder
-FROM golang:1.17 AS csl-build-env
+FROM golang:1.23 AS csl-build-env
```

Comment:
i got rid of air in favor of entr (similar in concept to watchexec), but we still should probably match this to the nix flake or other way around